pom.xml

```diff
@@ -1482,6 +1148,12 @@
                 <artifactId>commons-net</artifactId>
                 <version>3.3</version>
             </dependency>
+            <dependency>
+                <!-- managed from jts 1.1.1 and geotools 1.1 -->
+                <groupId>com.googlecode.json-simple</groupId>
```

Comment:
Ok, am I reading this correctly that GeoTools is the project on the older version?

Reply:
yeah, but it's just a bugfix release so hopefully will be ok